src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -0,0 +1,34 @@
+import { NgModule } from '@angular/core';
+import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+
+
+@NgModule({
+    imports: [AuthModule.forRoot({
+        config: {
+            triggerAuthorizationResultEvent: true,
+            postLoginRoute: '/welcomeuser',
+            logLevel: LogLevel.Debug,
+            historyCleanupOff: true,
+            eagerLoadAuthWellKnownEndpoints: false,
+            authority: 'https://dev.oidc.gov.bc.ca/auth/realms/onestopauth-basic',
+            redirectUrl: 'https://jag-csrs-portal-public-f0b5b6-dev.apps.silver.devops.gov.bc.ca',
+            postLogoutRedirectUri: 'https://jag-csrs-portal-public-f0b5b6-dev.apps.silver.devops.gov.bc.ca',
+            clientId: 'jsb-csrs-991',
+            scope: 'openid profile',
+            //usePushedAuthorisationRequests: true, // use par Pushed Authorisation Requests
+            responseType: 'code',
+            silentRenew: true,
+            silentRenewUrl: window.location.origin + '/silent-renew.html',
+            useRefreshToken: true,
+            renewTimeBeforeTokenExpiresInSeconds: 10,
+            autoUserInfo: true,
+            customParamsAuthRequest: {
+              prompt: 'consent', // login, consent
```

Comment:
is there a hint we can use to avoid displaying all the login options and only use bceid basic?

Reply:
I would like to check